### DISCUSSION TITLE
compute hash dynamically (RAGE_JOAAT doubt)

### DISCUSSION THREAD

**Discussion by lonelybud** [YimMenu/YimMenu#1916] [Q&A] [answered]
is there any other way of computing hash dynamically bcz this does not work -
```c++
	rage::joaat_t objectHash = RAGE_JOAAT(selecObject);
```
where `selecObject` is a string that is not known at compile time

**Comment by Yimura** [comment, marked as answer]
rage::joaat() is the runtime function to get the hash from a string, RAGE_JOAAT is for compile time only.
